adazza-scrooge.gradle

```diff
@@ -0,0 +1,44 @@
+/*
+Apply this plugin at the root level of your finagle scala service
+to enable scrooge compilation for your *-api submodule.
+*/
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath group:'com.twitter',                  name:'scrooge-generator_2.12',          version:"$scroogeVersion"
+        classpath group:'com.github.iamsteveholmes',    name:'scrooge-gradle-plugin',           version:"$scroogeGradlePluginVersion"
+    }
+}
+
+configure(subprojects.findAll {it.name.endsWith('-api')}) {
+  apply from: 'https://raw.githubusercontent.com/adazza/adazza-gradle/master/adazza-scala.gradle'
+  apply plugin: org.iamsteveholmes.gradle.ScroogePlugin
+
+  compileScrooge {
```

Comment:
Due to some bit of gradle I don't yet understand running `gradle clean build` won't recompile  the scrooge files if the -api module lacks any scala files. I think it is b/c the scrooge tasks is link to compileScala. In practice shouldn't matter as we will likely have scala files in always.